.changeset/fast-cars-hide.md

```
@@ -0,0 +1,5 @@
+---
+"@spear-ai/ruff-config": major
+---
+
+Updated Ruff version to allow Python version inference and removed arbitrary Ruff Python version pin.  Fixed deprecated [tool.poetry] fields in pyproject.toml.  Ruff now uses the Python version specified in the pyproject.toml file.
```

markdownlint-cli2 (0.17.2)

First line in a file should be a top-level heading
null

(MD041, first-line-heading, first-line-h1)